docs/KUBERNETES.md

```diff
@@ -24,6 +24,14 @@ git checkout major.minor.patch-beta.n
 
 To view a list of all available release versions, navigate in your terminal to your local working directory (where you cloned the agent source repository) and then run the command `git tag -l`.
 
+__NOTE:__
+For a quick installation without cloning the GitHub repo, you can run the following three commands in your terminal to create a logdna-agent namespace with your ingestion key and then deploy the LogDNA Agent DaemonSet to your cluster.
+```bash
+kubectl apply -f https://assets.logdna.com/clients/agent-namespace.yaml
+kubectl create secret generic logdna-agent-key -n logdna-agent --from-literal=logdna-agent-key=<your ingestion key> # this is your unique Ingestion Key
+kubectl apply -f https://assets.logdna.com/clients/agent-resources.yaml`
```

Comment:
Note that `https://assets.logdna.com/clients/agent-resources.yaml` is pinned to version 2.2.4 of the agent.

To install the latest ones, the user should use a version, like the following:

`https://assets.logdna.com/clients/logdna-agent/3.2.0/agent-resources.yaml`

It would be probably a good idea to provide a generic 3.x version yaml. I don't know all the details of it, maybe @c-nixon can provide some more info.

Reply:
Ah thanks for catching that @jorgebay ! There is a ticket for getting a generic yaml... but in the meantime, let's use the 3.2 URL you provide here. 

@alfeng6 and @c-nixon is that OK by you two? Now that I better understand this pinning, I can help us remember to switch it to 3.3 when we GA. ;-)

We should also change in the in-app docs. I'll make a ticket.